.env.example

```
@@ -1 +1,28 @@
+# Copy this file to `.env` and fill in values as needed.
+# Notes:
+# - `.env` is local-only and gitignored; do not commit.
+# - Run `bun run validate-env` to sanity check, and `bun run env:diff` to compare keys with this file.
+# - CI provides GITHUB_TOKEN via GitHub Secrets for workflows; locally you can set it here.
+# - Ticket API is removed. Do NOT set TICKET_API_PORT.
+# - Dev server defaults to 5500; change only if necessary.
+
+# GitHub – used by scripts/learning/open-pr.js and tooling that talks to GitHub
+# PAT scope suggestion: repo (or fine-grained with pull requests + contents)
+GITHUB_TOKEN=
+GITHUB_OWNER=edwardfalk
```

dotenv-linter (3.3.0)

[warning] [UnorderedKey] The GITHUB_OWNER key should go before the GITHUB_TOKEN key

(UnorderedKey)